test/profiling/MemoryTestPublisher.cpp

```diff
@@ -456,44 +449,40 @@ bool MemoryTestPublisher::test(
     {
         for (unsigned int count = 1; count <= n_samples; ++count)
         {
-            if (dynamic_data)
+            if (dynamic_data_)
             {
-                m_DynData->set_uint32_value(count, 0);
-                mp_datapub->write((void*)m_DynData);
+                m_DynData->set_uint32_value(0, count);
+                std::cout << "writing" << std::endl;
```

Comment:
Do we want to keep these traces or were they added in order to debug the test? If the tests uses a static type then no trace is displayed.

Reply:
Done in d5af16ae0